README.md

```diff
@@ -264,6 +264,56 @@ module.exports = ({ a, b }) => {
 };
 ```
 
+### Broadcast a message to all worker threads
+
+Piscina supports broadcast communication via BroadcastChannel(Node v18+). Here is an example, the main thread sends a message, and other threads the receive message.
+
+In `main.js`
+```js
+'use strict';
+
+const { BroadcastChannel } = require('worker_threads');
+const { resolve } = require('path');
+
+const Piscina = require('piscina');
+const piscina = new Piscina({
+  filename: resolve(__dirname, 'worker.js'),
+  useAtomics: false
```

Comment:
Let's just add a comment of why `useAtomics` is disabled, and it should be ready to go

Reply:
Of course, it is necessary